src/ZKMRStakeRegistry.sol

```diff
@@ -184,7 +221,8 @@ contract ZKMRStakeRegistry is
             delegationManager.getOperatorShares(operator, strategies);
 
         for (uint256 i; i < strategyParams.length; i++) {
-            totalShares += shares[i] * strategyParams[i].multiplier;
+            totalShares +=
+                shares[i] * (withMultipliers ? strategyParams[i].multiplier : 1);
```

Comment:
I'm not sure i fully understand why we would want to not use the strategy multiplier. Could you write something in comments about when to use them or not ? Feels like this is not something we control, so that value seems important to take into consideration but i may miss something obvious there.